UPEI-Android/group-project-mindjogg
# Dev client folderstructure

## Source 

![image](https://user-images.githubusercontent.com/55472981/151683596-8a9ea0ff-6755-4907-8d6b-0f4168ec99c0.png)

![image](https://user-images.githubusercontent.com/55472981/151683605-a4e84479-f2cf-43a8-ba00-a3dde967c251.png)

## Description 

### Summary of PR 

The PR will remove app sub folder from mindjogg folder and Home icon of Meterial UI

### Purpose 

The are rendering error in Android and IOS app. 

## Review and Testing Steps

1. Pull the new branch 
2. Build the mindjogg app with ```npm start ```
2. Check if you see the error in Android and IOS emulator or Physical Android and IOS device 


If everything works then the code are fine otherwise let me know.

### Notes
We do not need "app" sub folder inside mindjogg folder. It contradicts with the name of whole app.
